wmn-data.json

```diff
@@ -2931,6 +2931,16 @@
         "known" : ["test"],
         "cat" : "gaming"
        },
+       {
+        "name": "komi",
+        "uri_check": "https://{account}.komi.io",
+        "e_code": 200,
+        "e_string": "Privacy Preferences",

```

Comment:
I'm not seeing this in the source of the response. It is loaded later through graphsql or other web calls but not after the initial request.